src/components/Experiment/QuickInput/Tabs/SampleInput/SampleInputsTab.js

```diff
@@ -76,6 +86,15 @@ export default function SampleInputsTab(props) {
         );
     }
 
+    function makeSampleVideoInput(url, index) {
+        return (
+            <button onClick={() => { inputHandlerForPreview(url.src); selectInput(index); }} key={index} className={getElement(getInputClassName(url))}>
```

Comment:
You should make a named onClick function above (like `const onSampelVideoInputClick`) that will call `inputHandlerForPreview` and `selectInput`, instead of calling them both here

Reply:
Yeah, that would be a better approach. Thanks, fixed it.